Dockerfile

```diff
@@ -1,12 +1,12 @@
-ARG BUILDER_IMAGE=golang:1.15.5
+ARG BUILDER_IMAGE=golang:1.15.9
```

Comment:
```suggestion
ARG BUILDER_IMAGE=golang:1.16.0
```

Reply:
will do in a follow up, have move things to do